src/scrabble.js

```diff
@@ -1,6 +1,22 @@
+const letterValue = { a: 1, b: 3, c: 3, d: 2, e: 1, f: 4, g: 2, h: 4, i: 1, j: 8, k: 5, l: 1, m: 3, n: 1, o: 1, p: 3, q: 10, r: 1, s: 1, t: 1, u: 1, v: 4, w: 4, x: 8, y: 4, z: 10 }
```

Comment:
Could have separate lines for each point value -> 1, 2, 3, 4, 5, 8, 10 for better readability here - for example: `twoPoints: ["d", "D", "g", "G"],`